packages/jest-util/src/__tests__/console.test.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import chalk from 'chalk';
+import CustomConsole from '../Console';
+
+describe('CustomConsole', () => {
+  let _console;
+  let _stdout = '';
+
+  beforeEach(() => {
+    _console = new CustomConsole(process.stdout, process.stderr);
+    // override the _logToParentConsole method to be able and assert on the stdout.
```

Comment:
You can use `jest.spyOn` instead, perhaps?

Reply:
Yes, indeed, thanks.